src/server/server_family.cc

```diff
@@ -109,6 +109,11 @@ string UnknownCmd(string cmd, CmdArgList args) {
                       StrJoin(args.begin(), args.end(), ", ", CmdArgListFormatter()));
 }
 
+// Replace "{timestamp}" with the current timestamp.
```

Comment:
update comment

Reply:
just removed it instead, wasn't very high value